tests/test_proxy_functional.py

```diff
@@ -250,6 +250,7 @@ async def test_https_proxy_unsupported_tls_in_tls(
 # Filter out the warning from
 # https://github.com/abhinavsingh/proxy.py/blob/30574fd0414005dfa8792a6e797023e862bdcf43/proxy/common/utils.py#L226
 # otherwise this test will fail because the proxy will die with an error.
+@pytest.mark.xfail  # TODO: Fix this test
```

Comment:
Plz always pass condition, reason and raises to xfail markers.

Reply:
I am not sure why the tests are failing, it seems there are ssl errors related to `aiohttp.io` but I am getting those even without this PR locally.

Reply:
@bdraco would probably have better understanding since he fixed this test in the past. It runs `proxy.py` on the background. I'd recommend checking if that project is compatible with Python 3.14 first.

Reply:
They appear to be passing in CI (which means the CI is failing now). So, I don't think these should be changed.

Reply:
I removed the xfail markers, so far the tests passed on 3.11 and above but is failing on 3.10 and below.